src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -894,24 +894,24 @@ private StringBuilder getFeedbackSessionResultsForQuestionInCsvFormat(
             
             // keep track of possible recipients with no responses
             removeParticipantIdentifierFromList(question.giverType,
-                    possibleGiversWithoutResponses, response.giverEmail, fsrBundle);
+                    possibleGiversWithoutResponses, response.giver, fsrBundle);
             
-            boolean isNewGiver = !prevGiver.equals(response.giverEmail);
+            boolean isNewGiver = !prevGiver.equals(response.giver);
             // print missing responses from the current giver
             if (isNewGiver) {
                 exportBuilder.append(getRowsOfPossibleRecipientsInCsvFormat(fsrBundle,
                         question, questionDetails,
                         possibleRecipientsForGiver, prevGiver));
                 String giverIdentifier = question.giverType == FeedbackParticipantType.TEAMS
-                                       ? fsrBundle.getFullNameFromRoster(response.giverEmail)
-                                       : response.giverEmail;
+                                       ? fsrBundle.getFullNameFromRoster(response.giver)
+                                       : response.giver;
```

Comment:
Maybe indent these 2 lines 2 spaces to the right as well? It seems to be the convention now that `?` and `:` are not aligned with `=` for assignment.